packages/rfsm/rfsm.1.7.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis:
+  "A toolset for describing and simulating StateChart-like state diagrams"
+description:
+  "A toolset for describing and simulating StateChart-like state diagrams"
```

Comment:
Side note: the description is the same as the `synopsis`. It would be nice to either remove the `description` field or writing something a bit more substantial for the description.